models/token.go

```diff
@@ -77,6 +77,12 @@ func GetAccessTokenBySHA(token string) (*AccessToken, error) {
 	return nil, ErrAccessTokenNotExist{token}
 }
 
+// AccessTokenByNameExists checks if a token name has been used already by a user.
+func AccessTokenByNameExists(token *AccessToken) bool {
+	exists, _ := x.Table("access_token").Where("name = ?", token.Name).And("uid = ?", token.UID).Exist()
```

Comment:
Error should not be ignored and returned down to caller where status code 500 should be returned in case of error